kube/deploy.sh

```diff
@@ -10,13 +10,6 @@ echo "Deploying hocs-info-service to ${ENVIRONMENT}"
 echo "Service version: ${VERSION}"
 echo "Data version: ${HOCS_INFO_SERVICE_DATA_VERSION}"
 
-if [[ ${KUBE_NAMESPACE} =~ w?cs-(qa|demo|prod) ]]; then
-    if ! [[ ${VERSION} == "${HOCS_INFO_SERVICE_DATA_VERSION}" ]]; then
-        echo "Service and data versions don't match for namespace: ${KUBE_NAMESPACE}" > /dev/stderr
```

Comment:
Is there any value to keeping this as a warning?

Reply:
Would people actually look at it? As it defaults to the same - you have to intentionally change it?